articles/aks/open-ai-quickstart.md

```diff
@@ -408,6 +411,8 @@ Now that you've seen how to add OpenAI functionality to an AKS application, lear
 
 [managed-identity]: /azure/ai-services/openai/how-to/managed-identity#authorize-access-to-managed-identities
 
+[Create a private AKS cluster]: private-clusters.md?tabs=azure-portal#options-for-connecting-to-the-private-cluster
```

Comment:
You can remove the other parameters after the name of the article - just keep private.cluster.md and strip off the tail.